requirements/requirements.in

```diff
@@ -45,6 +45,7 @@ prometheus_client
 psycopg
 psutil
 pygerduty
+PyGithub <= 2.6.0  
```

Comment:
@thedoubl3j it's a transitive dep. Isn't this file supposed to list direct deps? The correct thing to do would be to depend on `awx-plugins-core [credentials-github-app]` that declares the dependency.

Reply:
You are correct. Put up https://github.com/ansible/awx/pull/15853

I was a bit panicked to merge this PR yesterday, because it was breaking checks and presumably the installs. If we have an outright missing dependency, I'd hope that we could ignore _all checks_ and force it through. After that, fine to do cleanup.

The way this is a floating dependency on awx-plugins-core, honestly, our best-case scenario would have been to add it to `requirements.in` a day before merging the plugins patch. Then merge the plugins patch, then update the requirements. Otherwise - pin the plugins version.